src/ai/backend/manager/api/session.py

```diff
@@ -1453,7 +1467,9 @@ async def report_stats(root_ctx: RootContext) -> None:
 async def stats_report_timer(root_ctx: RootContext):
     while True:
         try:
-            await asyncio.shield(report_stats(root_ctx))
+            aiotools.create_timer(
```

Comment:
`aiotools.create_timer()` should NOT be used inside a while loop, because it creates a coroutine that runs a loop with sleeps.
Rewrite `stats_report_timer()` as a one-shot execution function and put it inside the timer, instead of creating timer whenever it executes.  This is a memory/task leak.